KeenClient/KeenLogger.m

```diff
@@ -54,13 +54,13 @@ - (KeenLogger*)init {
         // about concurrent access when adding and removing loggers
         // since those operations will be dispatched to this queue as well.
         _loggerQueue = dispatch_queue_create("io.keen.logger", DISPATCH_QUEUE_SERIAL);
-        if (NULL == _loggerQueue) {
+        if (_loggerQueue == NULL) {
```

Comment:
I wouldn't... but that's mainly a stylistic thing with me, I find ```if (!_loggerQueue)``` harder to read. If you want me to do that though, I totally can.

In general I tried to keep the existing style as much as possible, and simply clean up inconsistencies.